MySolution.rb

```diff
@@ -0,0 +1,36 @@
+class MySolution
+
+    #problem1
+    def sumOfEven(array)
+        array.select{|v| v.even?}.sum
+    end
+    
+    #problem2
+    def getLargest2Numbers(array)
+        bigMax = -1/0.0
+        smallMax = -1/0.0
+        for i in array
+            if i > bigMax and i > smallMax 
+                smallMax = bigMax
+                bigMax = i  
+            else
+                if i > smallMax 
+                    smallMax = i 
+                end
+            end
+        end
+        result = [smallMax, bigMax]
+        return result
```

Comment:
Do you need a variable named result here?
Do you need to explicitly specify `return` in ruby? (this was a previous question from @alexandru-calinoiu )